extensions/3DTILES_implicit_tiling/0.0.0/schema/subtree.schema.json

```diff
@@ -2,37 +2,39 @@
   "$schema": "http://json-schema.org/draft-04/schema",
   "title": "Subtree",
   "type": "object",
-  "description": "An object describing the availability of tiles, children, metadata, layers, and content in a subtree.",
+  "description": "An object describing the availability of tiles and content in a subtree, as well as availability of children subtrees.",
   "properties": {
     "buffers":{
       "type": "array",
-      "description": "An array of buffers storing binary availability metadata.",
+      "description": "An array of buffers.",
```

Comment:
Trimmed down the description because bufferViews will eventually also store tile metadata and other per-subtree "stuff" that isn't necessarily availability.